host-bin/edit-chroot

```diff
@@ -335,15 +363,34 @@ for NAME in "$@"; do
                 dest="$dest.gz"
             fi
         fi
+        # If we're writing to a fat32 filesystem, split the file at 4GB chunks
+        if ! df -T "`getmountpoint "$dest"`" | awk '$2~"^v?fat"{exit 1}'; then
+            SPLIT="${SPLIT:-4GB}"
```

Comment:
Please add a comment that `4GB = 4 * 10^9` (and not `4 * 1024^3` (4G), since the fat32 limit is 4G-1 bytes). Actually, I'd almost prefer `4000000000` as GB is not POSIX.